### ISSUE TITLE
twitch.tv videos do not load

### ISSUE THREAD

**Issue by charbelrami** [brave/browser-laptop#1546]
twitch.tv videos do not load. They load in Chrome. It might be a local problem, because they load in other locations too.
Brave version: 0.9.3 RC3
Related: https://github.com/brave/browser-laptop/issues/1325

**Comment by diracdeltas** [comment]
they're working for me on 0.9.3 so far. example video? and which OS?

**Comment by bbondy** [comment]
ditto works for me.

**Comment by charbelrami** [comment]
windows 10, any stream video

**Comment by luixxiul** [comment]
They load on Windows 10, except one on the home page.

![](https://cloud.githubusercontent.com/assets/3362943/15271747/24f55b86-1a93-11e6-9878-2dec3df786cb.jpg)

I am not sure whether this is because no stream is available. Would you please check how it is on OS X, @bbondy @diracdeltas ?

**Comment by luixxiul** [comment]
https://github.com/brave/browser-laptop/wiki/Flash-and-Other-Non-Supported-Plugin-Sites-(Video-and-Non-Video)#brave-hack-to-use-html5-video

**Comment by luixxiul** [comment]
ref: #1308

**Comment by bbondy** [comment]
I think maybe live twitch streams do not play, but I think there was a more general problem for @charbelrami.  The live stream one should probably be a different issue.

**Comment by cezaraugusto** [comment]
Could someone confirm if it's still an issue?

**Comment by poga** [comment]
all twitch videos/live streams doesn't load for me. 
example video: https://www.twitch.tv/gogcom/v/78565380
![2016-07-18 3 08 51](https://cloud.githubusercontent.com/assets/8631/16902565/0a5725aa-4c95-11e6-8c8f-3e2b30b99bf9.png)

Brave: 0.11.0
OS: OS X 10.11.5

however, from dev tool I can see there's a lot of video-related request
![2016-07-18 3 13 46](https://cloud.githubusercontent.com/assets/8631/16902578/c54b2226-4c95-11e6-9064-53cbc0771089.png)

not much else interesting in the dev tool console.

**Comment by Mhynlo** [comment]
I just started using Brave, and I came across this issue with watching any of the twitch video streams on the home page. I am noticing that the individual streams are viewable, and in some rare cases the streams are viewable on the home page.
Brave Version 0.12.15
Windows 10

I clicked around the video settings related to the html5, but never saved.... No idea how it was starting to work again. Not sure if there was a cache that kicked in or something.

Not sure if this is common for other users still or if there is a setting that I am over looking.
